examples/basic/lightbulb/src/App.tsx

```diff
@@ -0,0 +1,22 @@
+import { Component, createSignal, Show } from 'solid-js';
+
+import styles from './App.module.css';
+import bulbOff from './assets/light-bulb-off.jpg';
+import bulbOn from './assets/light-bulb-on.jpg';
+
+const App: Component = () => {
+  const [lightOn, setLightOn] = createSignal(false);
+
+  return (
+    <>
+      <h1 class={styles.header}>Click the light to turn it on / off</h1>
+      <div class={styles.bulbContainer} onClick={() => setLightOn((lightOnValue) => !lightOnValue)}>
+        <Show when={lightOn()} fallback={<img src={bulbOff} />}>
+          <img src={bulbOn} />
```

Comment:
This would be more efficient without `<Show>`:

```jsx
        <img src={lightOn() ? bulbOn : bulbOff} />
```

This avoids DOM creation/deletion/mounting. But sadly no longer illustrates `<Show>`. 😅

Reply:
I think it would be nicer if the lightbulb was SVG and an additional <g>roup of paths to light it up was put inside <Show>.

Reply:
So, there is a lot more examples to come where we can better demo `<Show>` and Im all for doing this here

```jsx
        <img src={lightOn() ? bulbOn : bulbOff} />
```

But I'm hesitant to do the SVG route. The idea for this demo is just something really basic where a user can come to as a reference to figure out how to track state in Solid while cutting out as much noise as possible in the code backing the example.